hms-app/src/components/card-details/CardButton.tsx

```diff
@@ -14,6 +14,7 @@ import MoveIdeaModal from '../MoveIdeaModal'
 type IProps = {
   idea: Idea
   reloadIdeaDetails?: () => void
+  reloadIdeaList: () => void
```

Comment:
should also be nullable here then

Reply:
these seem like a pretty similar scenario to the previous PR I was looking at that where these could be named a bit differently, here I would say they could be `onIdeaEdited` and `onIdeaDeleted`.

This whole file could/should also probably be named differently, it is called `CardButton` but appears to be specific to an Idea and is likely something more like `IdeaCardButtonGroup` or so

Reply:
I would even be tempted to say this should be split into two things, a `DeleteIdeaButton` and an `EditIdeaButton` and then instead of having this `CardButtons`, just using those to components directly in the `IdeaDetails` 🤷

Reply:
but if you always use the edit and delete buttons as a pair then this makes sense as it is

Reply:
1. yeah similar case and we already opened a ticket for the button/idea files but will only tackle them after the refactoring is done so that we dont create more conflicts then nessesary
2. splitting it even more would be not a bad idea but yeah right now we always use these buttons in combination